Assignment1/TripletSum.cpp

```diff
@@ -0,0 +1,56 @@
+#include "TripletSum.h"
+
+// The Following Algorithm has been used:
+// Sort the array and iterate the array from left to right
+// For each Current Element of the array Try to find the rest of the
+// Two Elements using Two Pointer Technique. 
+
+bool TripletSum::checkTargetSum()
+{
+    // return false if the size is less than 3
+    if ( data.size() < 3 )
+    {
+        return false;
+    }
+	
+    sort(data.begin(), data.end());
+
+    // Iterate the vector from light to right
+    for ( int fixedIndex = 0; fixedIndex < data.size() - 2; ++fixedIndex )
+    {
+        int leftIndex = fixedIndex + 1, rightIndex = data.size() - 1;
+
+        // Sum might overflow, so use long long
+        long long int currentSUm = data[fixedIndex] + data[leftIndex] + data[rightIndex];
```

Comment:
you seem to be using camel casing convention of variable names. Thats fine but in that spirit currentSUm should be currentSum.